src/templates.rs

```diff
@@ -316,6 +317,7 @@ impl Templater {
                 o.insert(name_std.clone(), name);
 
                 match schema {
+                    Schema::Ref { name: _ } => todo!("Schema::Ref"),
```

Comment:
Quick and dirty solution!
I will have to take a deeper look to see what would be a better one for each case below.